test/type/union_test.exs

```diff
@@ -81,6 +81,11 @@ defmodule Ash.Test.Type.UnionTest do
           )
         end
       end
+
+      update :add_things do
+        require_atomic? false
```

Comment:
@zachdaniel the rebase from the original version required me to add this. TBH I'm not sure why 🤷‍♂️